src/math/p5.Vector.js

```diff
@@ -2355,35 +3101,60 @@ p5.Vector = class {
   // Static Methods
 
   /**
- * Make a new 2D vector from an angle.
+ * Create a new 2D vector from an angle.
```

Comment:
Should this be "creates" to have consistent tense with the others above (e.g. "returns ....")